### ISSUE TITLE
Cannot find the SVG renderer. Why is it missing?

### ISSUE THREAD

**Issue by Emasoft** [threeDart/three.dart#134]
Cannot find the three.js svg renderer (SVGRenderer.js) in your Dart port. 
In the original three.js repo it's here:

https://github.com/mrdoob/three.js/blob/master/examples/js/renderers/SVGRenderer.js

**Comment by joelday** [comment]
Agreed that it is a valuable example of a renderer that targets a persistent graph.
